moveit_ros/planning/planning_scene_monitor/src/planning_scene_monitor.cpp

```diff
@@ -263,7 +204,45 @@ void PlanningSceneMonitor::initialize(const planning_scene::PlanningScenePtr& sc
   state_update_timer_ =
       node_->create_wall_timer(dt_state_update_, std::bind(&PlanningSceneMonitor::stateUpdateTimerCallback, this));
 
-  // reconfigure_impl_ = new DynamicReconfigureImpl(this);
+  // Set up publishing parameters
+  bool publish_planning_scene = node_->get_parameter("planning_scene_monitor.publish_planning_scene").as_bool();
```

Comment:
I think the new parameters should be declared with default values (and possibly description) just copied from cfg file. These should be optional like before, currently the demo fails with a `ParameterNotDeclaredException`.
```suggestion
bool publish_planning_scene = node_->declare_parameter("planning_scene_monitor.publish_planning_scene", false);
```

Reply:
Also, what about `publish_planning_scene_hz`?

Reply:
switch to declare cdd51ad 

add `publish_planning_scene_hz` 2cd4f3b